string machine_name
---
ObjRefMsg   this_machine
string      this_node_name
ObjRefMsg[] other_machines